client/components/Navbar.js

```diff
@@ -30,23 +30,45 @@ const NavBar = props => {
     let firstLetter = petType.type[0].toUpperCase();
     let restOfWord = petType.type.slice(1);
     let typeOfPet = firstLetter + restOfWord;
-    return <Link to={`/pets/${petType.type}`} key={petType.id}>{typeOfPet} </Link>
+    return(
+      <li key={petType.id}>
+        <Link to={`/pets/${petType.type}`} >{typeOfPet} </Link>
+      </li>
+    ) 
   })
-
+  
   return(
     <div>
+      <div id="top-header">
+        <div>
+          {/* <img src={require("../assets/images/logo.png")} alt="logo" /> */}
```

Comment:
remove if not used!

Reply:
Serge may add this later, but he'll make a new branch if he does.